src/Open3D/Visualization/Rendering/Filament/TriangleMeshBuffers.cpp

```diff
@@ -41,6 +34,12 @@
 
```

Comment:
delete empty line?

Reply:
clang-format inserted the blank line. check-style fails without it.

Reply:
this turns out to be a difference between the way clang-format-5 and clang-format-9 implement SortIncludes. Empty line removed.